### DISCUSSION TITLE
使用FlinkStream方式从mysql同步到另一个mysql，无法完成数据同步。

### DISCUSSION THREAD

**Discussion by StoneMa** [apache/flink-cdc#2563] [Q&A]
程序如下：
![image](https://github.com/ververica/flink-cdc-connectors/assets/6917909/d9da9b50-76ed-4cc1-84ea-fb7e62a08808)
![588baf5078993d2cd013b98720ca0ec](https://github.com/ververica/flink-cdc-connectors/assets/6917909/3f98cb6a-7e4d-4957-9222-9046b0d01b50)

程序一直在运行，但是source库中的tests表中的数据，无法同步到target库的testt表中
![image](https://github.com/ververica/flink-cdc-connectors/assets/6917909/9246f71d-54b9-4ec9-af21-b647f100bf22)

mysql binlog已经开启

**Comment by StoneMa** [comment]
invoke方法没有被调用，请问是哪里的问题呢？

**Comment by gtk96** [comment]
@StoneMa 你试一下这个  demo 是 mysql to mysql 的示例，我本地测试是 ok 的https://github.com/SophiaData/Bigdata_Code_Tutorial/blob/master/sync_database_mysql/src/main/java/io/sophiadata/flink/sync/FlinkSqlWDS.java
